common/beerocks/bcl/include/bcl/beerocks_socket_thread.h

```diff
@@ -97,9 +97,10 @@ class socket_thread : public thread_base {
     int server_max_connections;
     SocketSelect select;
 
-    uint32_t m_select_timeout_msec                              = 0;
-    bool m_skip_next_select_timeout                             = false;
-    std::chrono::steady_clock::time_point m_select_wake_up_time = std::chrono::steady_clock::now();
+    uint32_t m_select_timeout_msec  = 0;
+    bool m_skip_next_select_timeout = false;
+    std::chrono::steady_clock::time_point m_select_wake_up_time =
+        std::chrono::steady_clock::time_point::min();
```

Comment:
I'm not sure I understand the problem and fix, but this feels like a bit of a hack

Reply:
It is not a hack, it more correct initialization value. I'll try to explain.
There is code on the work() that check how much time it took the thread to do a single select cycle and print a warning message if it exceeds the select timeout value.
In the fronthaul, we create the monitor thread sooner, but we don't start the thread right away. So on the first cycle of the work function it will print a warning message.
This happens since I have chosen to use `now()`, and we get delta time which exceeds the select_timeout.
Choosing `min()` cause to the delta to be negative and thus the condition for select_timeout result in "false" and the error is not printed.